### PR TITLE
Comment out typography import and typography property

### PR DESCRIPTION
**Pull request by jgabitto**
- Tees up https://github.com/cityofaustin/atd-data-tech/issues/6713
- Comment out typography import
- Comment out typography property in theme object

### CODE REVIEW

File: moped-editor/src/theme/index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const theme = createMuiTheme({
     },
   },
   shadows,
-  typography,
+  // typography,
```
Comment (sergiogcx) [inline moped-editor/src/theme/index.js:30]: If topography does not need to be there, then best to remove the comments too.

**Comment by sergiogcx** [comment]
@jgabitto @amenity does this seem right? The title looks disproportionally large:

**Comment by amenity** [comment]
@jgabitto - yep, it's tough to stomach but this is what I asked for.

**Comment by amenity** [comment]
Let's keep this open until we get other typography issues lined up to resolve it.

File: moped-editor/src/theme/index.js (hunk 1 above)
Comment (mateoclarke) [reply moped-editor/src/theme/index.js:30]: agreed

**Review by mateoclarke — COMMENTED** [review]
I agree with Sergio, instead of commenting out a line of code, you can always just delete it. If you need the old code back, it lives in the git history. That's the beauty of version control. Otherwise you end up with a bunch of edited out lines of code that you're not sure why they are there years down the line.

**Comment by amenity** [comment]
Just want to reiterate — this should not be merged into master!

**Comment by amenity** [comment]
In fact I think the PR can be closed — what we really wanted was the screenshots.

**Pull request closed without merging**